recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -57,6 +58,9 @@ PACKAGECONFIG ?= " \
 PACKAGECONFIG[bluetoothcontrol] = "-DPLUGIN_BLUETOOTH=ON -DPLUGIN_BLUETOOTH_AUTOSTART=true,-DPLUGIN_BLUETOOTH=OFF,,bluez5"
 PACKAGECONFIG[bluetoothremote]  = "-DPLUGIN_BLUETOOTHREMOTECONTROL=ON -DPLUGIN_BLUETOOTHREMOTECONTROL_AUTOSTART=true,-DPLUGIN_BLUETOOTHREMOTECONTROL=OFF,"
 
+PACKAGECONFIG_append_rpi = "cobalt"
+PACKAGECONFIG_append_brcm = "cobalt"
```

Comment:
Please place this directly under the PACKAGECONFIG ?= line and above the individual package config options. Else it gets quite messy.

Reply:
Changed the order